analytical_engine/java/grape-jdk/src/main/java/com/alibaba/graphscope/fragment/adaptor/ArrowProjectedAdaptor.java

```diff
@@ -0,0 +1,185 @@
+package com.alibaba.graphscope.fragment.adaptor;
+
```

Comment:
what is the purpose of adaptors?

Reply:
Now we have two Simple Fragment in `GRAPE-JDK`, i.e. `ArrowProjectedFragment`, `ImmutableEdgecutFramgnet`.
The underlying implementation is different, but they share the same Interface, So I created new Interface `SimpleFragment`, and adapt `Arrowprojected` and `ImmutableFragment` with corresponding adaptors.

The Reason why we don't replace all with one interface is that, the actual code is generated by FFI, and we Can not map One FFPointer with two C++ classes at the same time.